app/code/Magento/Review/Api/Data/ReviewInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Review\Api\Data;
+
+/**
+ * @api
+ * @since 100.0.2
+ */
+interface ReviewInterface
+{
+    // TODO: implement methods; currently used only as support for GraphQl
```

Comment:
Please define all methods. We need to have them before the initial release because an `@api` cannot have methods removed or added after that.

Reply:
defined in https://github.com/magento/magento2/pull/19266